pkg/api/api_default_service.go

```diff
@@ -796,25 +796,25 @@ func (cfm *CfmApiService) BladesPost(ctx context.Context, applianceId string, cr
 
 	appliance, err := manager.GetApplianceById(ctx, applianceId)
 	if err != nil {
-		return formatErrorResp(ctx, err.(*common.RequestError))
+		return openapi.Response(http.StatusBadRequest, nil), err
```

Comment:
All the error handling changes in this file are not acceptable.  When you hardcode error codes like this, you hide any other error codes that can occur.
What is the reason for this change?